shared/actions/chat2/index.tsx

```diff
@@ -1482,11 +1482,7 @@ const onInboxSearchSelect = (state, action: Chat2Gen.InboxSearchSelectPayload) =
   if (!conversationIDKey) {
     return
   }
-  const query = action.payload.query
-    ? action.payload.query // Auto generated from flowToTs. Please clean me!
-    : selected === null || selected === undefined
-    ? undefined
-    : selected.query
+  const query = action.payload.query ? action.payload.query : selected && selected.query
```

Comment:
This isn't right, needs parens after `:`

Reply:
o hang on, I think this is right (prettier changes it)
```
true ? 'a' : 'foo' && 'bar' // => 'a'
false ? 'a' : 'foo' && 'bar' // => 'bar'
```

Reply:
Hmm I guess it is, I tried something earlier in the console and thought it wasn't, but trying again now it seems like it's fine. Sorry!